docs/source/autodocs/eli5.rst

```diff
@@ -13,3 +13,5 @@ The following functions are exposed to a top level, e.g.
 .. autofunction:: eli5.show_weights
 
 .. autofunction:: eli5.show_prediction
+
+.. autofunction:: eli5.transform_feature_names
```

Comment:
Yeah, I was also unsure; do you think this function will be used stand-alone, not as an implementation detail of how to make explain.. / show.. functions work?